examples/semantic_segmentation/eval_semantic_segmentation.py

```diff
@@ -52,16 +58,22 @@ def get_dataset_and_model(dataset_name, model_name, pretrained_model,
     elif model_name == 'segnet':
         model = SegNetBasic(
             n_class=n_class, pretrained_model=pretrained_model)
+    elif model_name == 'deeplab_v3plus_xception65':
+        model = DeepLabV3plusXception65(
+            n_class=n_class,
+            pretrained_model=pretrained_model,
+            min_input_size=input_size)
```

Comment:
@yuyu2172  How do you feel using `input_size` as `min_input_size` in `eval_semantic_segmenation.py` like this

Reply:
or add description here
https://github.com/chainer/chainercv/pull/757/files#diff-b74057e88e45cdf3c2cb434f11678d80R79
```
parser.add_argument('--input-size', type=int, default=None,
                                     help='Input image size passed to the network. When DeepLab v3+, this is passed to `min_input_size`')
```

Reply:
I think this is ok